packages/rum-core/src/domain/rumSessionManager.ts

```diff
@@ -17,6 +17,7 @@ export interface RumSessionManager {
   findTrackedSession: (startTime?: RelativeTime) => RumSession | undefined
   expire: () => void
   expireObservable: Observable<void>
+  updateSessionInStore: (state: Partial<SessionState>) => void
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ I'm thinking it would be safer to expose something more explicit such as `setForceReplay()` or `setTrackingType()`. That way we avoid leaking implementation details and let the rum session manager in control of the the correct session key (i.e. `RUM_SESSION_KEY`)

i.e. it should not be possible to do something like `rumSessionManager.updateSessionInStore({ logs: RumTrackingType.TRACKED_WITH_FORCED_REPLAY })`

Reply:
Done ✔️